Userland/Libraries/LibWeb/CSS/StyleProperties.h

```diff
@@ -76,8 +76,8 @@ class StyleProperties {
         No,
         Yes,
     };
-    NonnullRefPtr<CSSStyleValue const> property(CSS::PropertyID, WithAnimationsApplied = WithAnimationsApplied::Yes) const;
-    RefPtr<CSSStyleValue const> maybe_null_property(CSS::PropertyID) const;
+    CSSStyleValue const& property(CSS::PropertyID, WithAnimationsApplied = WithAnimationsApplied::Yes) const;
+    CSSStyleValue const* maybe_null_property(CSS::PropertyID) const;
```

Comment:
We generally try to avoid raw pointers, but I think it's fine here because the `maybe_null_property()` name makes it clear enough.